From 07625a74a0d3d0ef859e75b5d3fe883bdfd579aa Mon Sep 17 00:00:00 2001
From: jeremyb31 <jeremy53561@gmail.com>
Date: Tue, 15 Feb 2022 16:18:15 -0600
Subject: [PATCH] Update btusb.c

---
 btusb.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/btusb.c b/btusb.c
index 71286ce..caaacd7 100644
--- a/drivers/bluetooth/btusb.c
+++ b/drivers/bluetooth/btusb.c
@@ -451,6 +451,10 @@ static const struct usb_device_id blacklist_table[] = {
 	{ USB_DEVICE(0x13d3, 0x3458), .driver_info = BTUSB_REALTEK },
 	{ USB_DEVICE(0x13d3, 0x3461), .driver_info = BTUSB_REALTEK },
 	{ USB_DEVICE(0x13d3, 0x3462), .driver_info = BTUSB_REALTEK },
+
+	/* Additional Realtek 8761B Bluetooth devices */
+	{ USB_DEVICE(0x2357, 0x0604), .driver_info = BTUSB_REALTEK |
+						     BTUSB_WIDEBAND_SPEECH },
 
 	/* Additional Realtek 8822BE Bluetooth devices */
 	{ USB_DEVICE(0x13d3, 0x3526), .driver_info = BTUSB_REALTEK },
